contrib/scripts/GOGconf-cp437-to-UTF-8.bat

```diff
@@ -0,0 +1,27 @@
+@ECHO OFF

```

Comment:
Add an SPDX and copyright comment at the top clarify the file's license.

Use `scripts/count-pvs-bugs.py` as an example.

Suggest a bit more explanation about the tool, and can mentioning that the primary use of this is to convert GOG conf files, as an example use case.

Also suggest clarifying that this is .BAT file is meant to be run on the Windoes host, and not insideDOSBox.